packages/app/app.py

```diff
@@ -23,11 +22,18 @@ def fetch_yelp_data():
     # Get request response. Set timeout to stop requests from waiting after 5 seconds
     response = requests.get(url, params=params, headers=headers, timeout=5)
 
-    if response.status_code == 200:
+    if response.status_code >= 500:
+        print('[!] [{0}] Server Error: Something is wrong with Yelp'.format(response.status_code))
+    elif response.status_code == 404:
+        print('[!] [{0}] URL Not Found'.format(response.status_code,api_url))  
+    elif response.status_code == 401:
+        print('[!] [{0}] Authentication Failed'.format(response.status_code))
+    elif response.status_code == 400:
+        print('[!] [{0}] Bad Request'.format(response.status_code))
+    elif response.status_code == 200:
         data = json.loads(response.text)
```

Comment:
If a scheduled job fails, I recognize that handling the error should be contingent on what the error is. My thinking is that If the error is:

- Server Error w/Yelp API - Handle this by aborting current job(s) and then trying again indefinitely while app is running.
- With URL Not Found, Authentication Fail, Bad Request or Unknown errors w/API - Handle these by pausing jobs or aborting jobs because fixing errors while app is running is not possible.
- If it's a misfired job by the scheduler, I don't know if I should pause job and then restart or abort current job and then restart.